timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -254,7 +286,93 @@ def _fit(
         **kwargs,
     ) -> None:
         self._check_fit_params()
-        self.time_limit = time_limit
+
+        fine_tune_time_limit = time_limit // 2 if self.fine_tune else 0
```

Comment:
We don't need to do this by dividing in half explicitly. As long as the entire workflow respects the time limit, we can allow fine tuning to take as much time as it needs.

Reply:
We need some guardrails here since otherwise finetuning may consume 100% of the time limit, no time will be left for inference, and the Chronos model will just raise an exception when predicting/computing the val score.

I guess that allocating 50% of the time limit for fine-tuning is a reasonable value, but maybe there exist better ways to handle the overall time limit.

Reply:
I will keep it like this for now and we can discuss again.

Reply:
sounds good.